runtime/storage_test.go

```diff
@@ -8102,495 +7831,6 @@ func TestGetDomainStorageMapRegisterReadsForV1Account(t *testing.T) {
 	}
 }
 
-func TestGetDomainStorageMapRegisterReadsForV2Account(t *testing.T) {
```

Comment:
This is the same as `TestGetDomainStorageMapRegisterReads` above, but the `Check if account is v2` reads are now gone.